data_loaders/json_loader.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC 
+from data_loaders.data_loader import DataLoader
+import json
+
+class JsonLoader(DataLoader):
```

Comment:
I just had it like that because collecting data in a json file didn't really fit under any of the other categories...I can just put this code into the driver and run it there instead if that is better?

Reply:
Yeah I think that's a better option.